app/javascript/controllers/partner_tags_controller.js

```diff
@@ -4,6 +4,8 @@ export default class extends Controller {
 	static values = { permittedTags: [String] };
 
 	connect() {
+		console.log("PartnerTagsController.connect");
```

Comment:
Remove debug.

Reply:
Will do, there was one with and one without so will switch to both without instead.